data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -659,6 +662,7 @@
         [/message]
         [message]
             role=escapee sidekick 0
+            #po: Speaker of random gender talking to male unit
```

Comment:
This is wrong. Each string must have a fixed gender for the speaker; you need a `female_message` here.

Reply:
Although on second thoughts, I'm not 100% if it's actually needed here; I don't see any first-person pronouns.

Reply:
Arrrgh, outside of string freeze I'd definitely fix it.

However, it's an old unchanged string, already translated in 21 languages. I've just added a po hint to it and fixed what gender the unit being spoken to is. I could improve the hint to say that the unit being spoken to is the deserter, but don't think I should add the "female^..." variant during the freeze.

Reply:
Fairly sure there are languages where male-to-male and female-to-male are different, even without a first-person pronoun. OTOH, that makes me sure that I shouldn't change the string for 1.18.

Reply:
There's Korean sibling terms for example, which depend on the gender of both the speaker and the addressed. That's the only example I know of, but if there's one example, there's likely others too.

Reply:
The statement that it's translated in 21 languages is false. It's already fuzzy in those languages, because "trolls" changed to "orcs".